src/CommandingOfficers/Tech.java

```diff
@@ -107,11 +170,11 @@ public int getRepairPower()
 
     GameEventQueue abilityEvents;
 
-    TechdropAbility(Commander myCO, String abilityName, int abilityCost, int buff, int num, int abilityRange)
+    TechdropAbility(Commander myCO, String abilityName, double abilityCost, int buff, int num, int abilityRange)
     {
       super(myCO, abilityName, abilityCost);
 
-      // Create a COModifier that we can apply to Patch when needed.
```

Comment:
This patch patches Patch out from a patch of not-Patch

Reply:
Indeed.